src/test/java/uk/gov/justice/probation/courtcasematcher/restclient/model/courtcaseservice/CCSExtendedCaseTest.java

```diff
@@ -83,4 +84,22 @@ public void shouldMapFromCourtCaseAndKeepIdsIfPresent() {
         assertThat(actualFirstDefendant.getAwaitingPsr()).isEqualTo(firstDefendant.getAwaitingPsr());
         assertThat(actualFirstDefendant.getBreach()).isEqualTo(firstDefendant.getBreach());
     }
+
+    @Test
```

Comment:
Probably not caused by your change but I noticed that the source field isn't exercised by any of these tests in here.

Reply:
Good spot, I added in source afterwards so must have missed it in this test. Will add it in